Source/FakeItEasy/Creation/FakeOptions.cs

```diff
@@ -2,34 +2,72 @@ namespace FakeItEasy.Creation
 {
     using System;
     using System.Collections.Generic;
-    using System.Linq;
     using System.Reflection.Emit;
-    using FakeItEasy.SelfInitializedFakes;
+    using FakeItEasy.Core;
 
     internal class FakeOptions
     {
+        private readonly List<Type> additionalInterfacesToImplement;
+        private readonly List<Action<object>> fakeConfigurationActions;
+        private readonly List<CustomAttributeBuilder> additionalAttributes;
+        private FakeWrapperConfigurator wrapper;
+
         public FakeOptions()

```

Comment:
Minor point, but this constructor can be removed and replaced with field initializers. Won't hold up the PR.